websites/D/Discord SFX/presence.ts

```diff
@@ -70,6 +70,36 @@ presence.on("UpdateData", async () => {
 		];
 	}
 
+	if (pathname.startsWith("/users/")) {
+		presenceData.details = `Viewing ${
+			document.title.split(" Profile")[0].split(" | ")[1]
+		} profile`;
+
+		presenceData.buttons = [
+			{
+				label: `Published ${
+					document.querySelector<HTMLSpanElement>('[data-premid-id="Published"')
+						?.textContent
+				} Sounds`,
+				url: `https://discordsfx.com/users/${pathname.split("/")[2]}`,
+			},
+			{
+				label: `Received ${
+					document.querySelector<HTMLSpanElement>(
+						'[data-premid-id="Downloads Received"'
+					)?.textContent
+				} Downloads`,
+				url: `https://discordsfx.com/users/${pathname.split("/")[2]}`,
```

Comment:
Labels should be action items "View Sounds", etc. They should not be used to display stuff you [couldn't fit in other fields](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#presencets).